plugins/catalog-backend-module-logs/package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@backstage/backend-plugin-api": "workspace:^",
     "@backstage/plugin-catalog-backend": "workspace:^",
```

Comment:
oof this is unfortunate, we don't want to import backends from modules -but it's using an export that needs to be moved to the -node package to resolve it. Was that what you were intending to do?